pkg/kubelet/config/etcd.go

```diff
@@ -61,6 +61,10 @@ func (s *sourceEtcd) run() {
 	boundPods := api.BoundPods{}
 	err := s.helper.ExtractObj(s.key, &boundPods, false)
 	if err != nil {
+		if (tools.IsEtcdNotFound(err)) {
+			glog.V(4).Infof("etcd failed to retrieve the value for the key %q. Error: %v", s.key, err)
```

Comment:
It means that the key is not present in etcd.
Since this is not really an error, we are reducing the verbosity to Info instead of it being Error.